TradeItIosTicketSDK2Tests/FakeTradeItLinkedBrokerAccount.swift

```diff
@@ -12,7 +12,7 @@ class FakeTradeItLinkedBrokerAccount: TradeItLinkedBrokerAccount {
 
     }
     
-    override func getPositions(onSuccess: @escaping () -> Void, onFailure: @escaping (TradeItErrorResult) -> Void) {
+    override func getPositions(onSuccess: @escaping ([TradeItPortfolioPosition]) -> Void, onFailure: @escaping (TradeItErrorResult) -> Void) {
```

Comment:
Yes, it was done weeks ago but the test wasn't updated.

No, but the others don't translate so well as discussed last night with Positions for Equity/FX.